### DISCUSSION TITLE
Add descriptive window titles when using debug/run_multuple_instances

### DISCUSSION THREAD

**Discussion by kit-solent** [godotengine/godot-proposals#7141] [Editor]
When using debug/run_multuple_instances I find it hard to know which window is which debug session. I am making an online multiplayer game and my separate windows are giving different error messages but I don't know which errors are associated with the host and which with the client. 
<img width="199" alt="image" src="https://github.com/godotengine/godot-proposals/assets/85214318/bf1d1a4c-bb7e-4117-ab55-5ca6e1d4c39a">
<img width="113" alt="image" src="https://github.com/godotengine/godot-proposals/assets/85214318/20dbfe97-dd8f-4f77-8aaf-8c511f15c903">

If the window title was changed from `<Game Name> (DEBUG)` to `<Game Name> (DEBUG session 1)` then it would be easier to correctly interpret error messages.

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/godotengine/godot-proposals/issues/3357.
